samples/client/petstore/typescript-angular-v4/npm/package.json

```diff
@@ -35,6 +35,6 @@
     "typescript": "^2.1.5"
   },
   "publishConfig": {
-    "registry": "https://skimdb.npmjs.com/registry"
```

Comment:
Do you know where these non-related changes in the generated output comes from?

Reply:
that's a good question, I also noticed this unrelated change, but I guess this comes from a merged change to the master without running the bin/windows/typescript-angular-v4-with-npm.bat script.